.github/workflows/test_iree.yml

```diff
@@ -169,10 +191,28 @@ jobs:
             --log-cli-level=info \
             --timeout=1200 \
             --durations=0 \
-            --config-files=${SDXL_CONFIG_FILE_PATH}
+            --config-files=${SDXL_UNET_CONFIG_FILE_PATH}
+      
+      - name: "Running real weights SDXL vae decode tests"
+        id: vae
+        if: ${{ !cancelled() }}
+        run: |
+          source ${VENV_DIR}/bin/activate
+          pytest iree_tests/pytorch/models/sdxl-vae-decode-tank \
+            -rpfE \
+            -k real_weights \
+            --no-skip-tests-missing-files \
+            --capture=no \
+            --log-cli-level=info \
+            --timeout=1200 \
+            --durations=0 \
+            --config-files=${SDXL_VAE_DECODE_CONFIG_FILE_PATH}
 
       - name: "Running SDXL ROCm pipeline benchmark"
-        if: contains(matrix.name, 'rocm')
+        if: |
+          contains(matrix.name, 'rocm') &&
+          steps.prompt_encode.outcome == 'success' && 
+          steps.unet.outcome == 'success' && steps.vae.outcome == 'success'
```

Comment:
Ah, good idea. I was wondering how to get the "continue if some tests fail but not these specific steps" behavior.

In particular, I don't want to run test jobs if repo checkout, venv setup, and other similar steps failed.

Reply:
Yeah makes sense, we can add all the different job dependencies in another PR. Don't want to run the model testing if the azure download fails as well for example